### ISSUE TITLE
Option "startTimer"

### ISSUE THREAD

**Issue by calebelt** [rkusa/touch-dnd#59]
Great idea to add the timer option for draggable elements. This is useful if you want to make the click event or doubleclick on element.
`
$(element).draggable({startTimer: 200});
`

**Comment by rkusa** [comment]
Hi, thanks for you feedback! I'll have a look at both of your issues next week.

**Comment by rkusa** [comment]
Right, sounds like a good idea. However, I don't have a personal use case for this. But I am happily accepting a PR for this. Thanks for the feedback!

**Comment by calebelt** [comment]
Please look at the link http://webismy.work/cms/dnd.html. In the lower left corner click on the button to open the control panel modules. In the upper part there are orange filters the list of modules (sorry, I haven't done the translation into other languages). These filters can be dragged down into a black hole :) for removal.
I plan to do more dragged elements with click event and doublecklick.
